Sources/MeiliSearch/Tasks.swift

```diff
@@ -49,21 +41,34 @@ struct Tasks {
 
   // get all on client
   func getAll(
+    params: TasksQuery? = nil,
     _ completion: @escaping (Result<Results<Task>, Swift.Error>) -> Void) {
-      getAll(path: "/tasks", completion)
+      getAll(path: "/tasks", params: params, completion)
   }
 
   // get all on index
   func getAll(
     uid: String,
+    params: TasksQuery? = nil,
```

Comment:
since the prototype is the same as the next overloaded function, how are we sure that we enter this function only when `getAll` is called from `indexes` ?

Reply:
When we call: `self.tasks.getAll(params: params, completion)` in the line 202 (from Client.swift) we are redirected to the `public` `getAll` function in the line 43. Then inside this function, we call the `private` `getAll` function, which just receives the params and makes the request.


When we call `self.tasks.getAll(uid: self.uid, params: params, completion)` in the line 476 (from Indexes.swift) we are redirected to the `public` `getAll` function in the line 51 not the public function in the line 43.

Reply:
To prevent those kind of misunderstandings I will rename this function :)

Reply:
my bad I did not see they had different params